api/openapi-spec/swagger.json

```diff
@@ -4259,23 +4256,22 @@
     }
    },
    "v1.Port": {
-    "description": "Port repesents a port to expose from the virtual machine.\nDefault protocol TCP.\nDefault port is PodPort.",
+    "description": "Port repesents a port to expose from the virtual machine.\nDefault protocol TCP.\nDefault podPort is port.",
     "required": [
      "port"
     ],
     "properties": {
      "name": {
+      "description": "If specified, this must be an IANA_SVC_NAME and unique within the pod. Each\nnamed port in a pod must have a unique name. Name for the port that can be\nreferred to by services.\n+optional",
```

Comment:
is it optional in case of slirp? could we define slirp without any port?

Reply:
We only need to port number just like kubernetes does.
the name is not mandatory.